docs/reference/troubleshoot.md

```diff
@@ -21,7 +21,7 @@ indent: true
 ## Requested Resource Not Found
 
 If you use the kubectl Crossplane plugin to install a `Provider` or `Configuration`
-(e.g. `kubectl crossplane install provider crossplane/provider-aws:master`) and
+(e.g. `kubectl crossplane install provider crossplane/provider-aws:v1.7.0`) and
```

Comment:
I think this tag may need to remain unchanged since v1.7.0 is not valid for provider-aws

Reply:
ah, good catch! fixed.